core/handler/manager_server.go

```diff
@@ -225,7 +225,8 @@ func (h *handlerManager) SetDevice(ctx context.Context, in *pb_handler.Device) (
 		changedFields := dev.ChangedFields()
 		for _, changedField := range changedFields {
 			switch changedField {
-			case "AppKey", "UsedDevNonces", "UsedAppNonces": // Allow changing the AppKey.
+			case "AppKey", "UsedDevNonces", "DevAddr", "UsedAppNonces", "NwkSKey", "AppSKey": // Allow changing the AppKey for OTAA and the session data for ABP.
+			case "FCntUp":
```

Comment:
Is this a leftover from "just trying something" or is this intended to be a separate case?

Reply:
This is intended:
- In v2 there are no field masks, so you need to patch the full device.
- If the device sends an uplink between a `Get` and ` Set`, the Handler detects that you're trying to update the frame counter and fails.

Do you see a better fix ?

Reply:
Okay, fine with me, but let's add a quick comment. Can you do that?